examples/with-logger/renderer/next.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  webpack: (config, { isServer }) => {
+    if (!isServer) {
+      config.target = 'electron-renderer';
+    }
+
+    return config;
```

Comment:
Just to keep it as simple as possible.
```suggestion
    if (!isServer) config.target = 'electron-renderer';
    return config;
```

Reply:
Isn't this just flavor? I personally don't like one-line if's for several reasons

- harder to read
- requires more work and thought when extending
- prone to errors (remember the hearbleed bug?)

But as stated, this is just coding style. I adjusted mine to be extremely declarative and easy to read/maintain when I started working as lead developer and teacher.

Just my two cents